vendor/k8s.io/kubernetes/pkg/volume/glusterfs/glusterfs.go

```diff
@@ -476,9 +484,9 @@ func (r *glusterfsVolumeProvisioner) Provision() (*api.PersistentVolume, error)
 		return nil, err
 	}
 	r.provisioningConfig = *cfg
-
 	glog.V(4).Infof("glusterfs: creating volume with configuration %+v", r.provisioningConfig)
-	glusterfs, sizeGB, err := r.CreateVolume()
+	reqGid = gidMin + gidRandomizer.Int63n(gidMax)
```

Comment:
Checking against SCC isn't my concern. A pod automatically picking up a gid by virtue of using a particular PV makes sense to me, but the gids here should be able to be shaped by a cluster admin… limited to a range, etc